carla-carma-integration/carla-carma-agent/agent/bridge.launch

```diff
@@ -93,4 +93,28 @@
     <param name='max_steering_degree' type="double" value="$(arg max_steering_degree)"/>
   </node>
 
+  <!--
+    # route #
+    Set the vehicle route after localization.
+  -->
+  <node pkg='carla_carma_bridge' type='carma_carla_route' name='carma_carla_route' output='screen'>
+    <param name='role_name' type="str" value="$(arg role_name)"/>
```

Comment:
Not sure but do we need ego vehicle role name in `carma_carla_route`? Also, the `select_route` should be set here since there is a `get_param` method being called [here](https://github.com/usdot-fhwa-stol/carma-simulation/blob/81b8c891c1d440c383adcab1f1f5b4bec4a995b9/carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carma_carla_route#L32)

Reply:
Hmm, I guess that depends on if we're interacting with the CARLA ROSbridge topics right? If this node subscribes to the CARMA-side of those interfaces it shouldn't need the role name I think.